src/axom/spin/internal/linear_bvh/build_radix_tree.hpp

```diff
@@ -42,44 +42,42 @@ namespace internal
 namespace linear_bvh
 {
 //------------------------------------------------------------------------------
-//Returns 30 bit morton code for coordinates for
-// x, y, and z are expecting to be between [0,1]
+//Returns 30 bit morton code for coordinates point is expected to be between [0,1]
 template <typename FloatType, int Dims>
-static inline AXOM_HOST_DEVICE std::int32_t morton32_encode(const primal::Vector<FloatType, Dims>& point)
+static inline AXOM_HOST_DEVICE std::uint32_t morton32_encode(const primal::Vector<FloatType, Dims>& point)
```

Comment:
Another minor change -- the return type for these `morton_encode` functions should be unsigned since the `MortonIndexType` is unsigned.  

For other recent changes to this function, see https://github.com/LLNL/axom/pull/1611/files#diff-17b5f2106d61093b9348d431dc0e84ce494f4551984015540561e7ebf53c0e32

While looking at this again, I decided to use `axom::clampVal` instead of `fmin(fmax, ...)`